public/app/features/alerting/unified/api/ruler.ts

```diff
@@ -36,10 +36,12 @@ export function rulerUrlBuilder(rulerConfig: RulerDataSourceConfig) {
         params: params,
       };
     },
-    namespace: (namespace: string): RulerRequestUrl => ({
-      path: `${rulerPath}/${encodeURIComponent(namespace)}`,
-      params: Object.fromEntries(rulerSearchParams),
-    }),
+    namespace: (namespace: string): RulerRequestUrl => {
```

Comment:
there is no change here, right?

Reply:
Right, I changed back to how it was before (removing the explicit `return` statement)